packages/kbn-babel-code-parser/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@kbn/babel-code-parser",
+  "description": "babel code parser for Kibana",
+  "private": true,
+  "version": "1.0.0",
+  "main": "./target/index.js",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elastic/kibana/tree/master/packages/kbn-babel-code-parser"
+  },
+  "scripts": {
+    "build": "babel src --out-dir target",
```

Comment:
Not that it needs changed for this right now - but is there a reason we need to compile this code?

Reply:
Yeah I have one, I think when I was writing the parser, I have used a syntax not yet supported by the node version at that time.